tests/sentry/region_to_control/test_utils.py

```diff
@@ -0,0 +1,22 @@
+import contextlib
+from unittest.mock import patch
+
+from sentry.testutils.silo import exempt_from_silo_limits
+
+# TODO:  Can a service abstraction that allows default stubs + opt-out ability replace this?
+
+
+@contextlib.contextmanager
+def flush_audit_logs():
```

Comment:
It would be nice if, by default, in Control Silo mode, there was a stub implementation (similar to the mock) that can turned OFF in the tests where I actually do full integration testing (there are a couple of those).  Right now I'm basically using this decorator everywhere I want to TURN OFF the producer, but ideally a stub service would just have the producer off by default and instead OPT IN to the real implementation.